File: Agora-Signaling-Tutorial-iOS-Objective-C/Agora-Signaling-Tutorial/Models/AgoraSignal.m
//
//  AgoraSignal.m
//  Agora-Signaling-Tutorial
//
//  Created by Zhangji on 2018/11/16.
//  Copyright © 2018 Agora. All rights reserved.
//

#import "AgoraSignal.h"
#import "KeyCenter.h"

@implementation AgoraSignal
+ (AgoraAPI *)sharedKit {
    static AgoraAPI *kit = nil;
    static dispatch_once_t onceToken;
    dispatch_once(&onceToken, ^{
        kit = [AgoraAPI getInstanceWithoutMedia:[KeyCenter AppId]];
    });
    return kit;
}
@end

File: Agora-Signaling-Tutorial-iOS-Swift/Agora-Signaling-Tutorial/Models/AgoraSignal.swift
//
//  AgoraSignalKit.swift
//  Agora-Signaling-Tutorial
//
//  Created by ZhangJi on 04/12/2017.
//  Copyright © 2017 ZhangJi. All rights reserved.
//

import Foundation
import AgoraSigKit

struct AgoraSignal {
    static let Kit : AgoraAPI = AgoraAPI.getInstanceWithoutMedia(KeyCenter.AppId)
}
